File: Level1/35.최대공약수와최소공배수/35.최대공약수와최소공배수.py
def findGcd(a, b):
    while b!=0:
        temp = a%b
        a = b
        b = temp
    return a
def solution(n, m):
    answer = []
    gcd = findGcd(n, m)
    lcm = (n*m) / gcd
    answer.append(gcd)
    answer.append(lcm)
    return answer

File: Level1/35.최대공약수와최소공배수/35.최대공약수와최소공배수.java
class Solution {
    public int findGcd(int a, int b){
        while(b!=0){
            int temp = a%b;
            a = b;
            b = temp;
        }
        return a;
    }
    public int[] solution(int n, int m) {
        int[] answer = new int[2];
        int gcd = findGcd(n, m);
        int lcm = (n*m) / gcd;
        answer[0] = gcd;
        answer[1] = lcm;
        return answer;
    }
}